intro/Readme.md

```diff
@@ -1,29 +1,43 @@
-1) What are the three most essential skills that a beginner programmer needs to know?
```

Comment:
This file shouldn't be edited as part of this PR.  Where is the chapter_2/Readme.md changes?